### ISSUE TITLE
Desktop platforms support

### ISSUE THREAD

**Issue by ElderOrb** [Larpon/QtFirebase#121]
Taking into account firebase provides subset of API for desktop platforms, would be great if QtFirebase could provide it too.

**Comment by Larpon** [comment]
Good point. It's on my TODO already to try and enable desktop features and maybe drop the stub implementation completely now that the C++ SDK is open source.

**Comment by toby20130333** [comment]
any qt desktop app support?

**Comment by Larpon** [comment]
@toby20130333 not yet
